src/ceph-detect-init/Makefile.am

```diff
@@ -55,11 +55,13 @@ EXTRA_DIST += \
 
 ceph-detect-init-all: ceph-detect-init/virtualenv
 
+CEPH_DETECT_INIT_VIRTUALENV := .
```

Comment:
can we have just one environment variable? Something like `CEPH_BUILD_VIRTUALENVS` and then specify the actual virtualenv for ceph-detect-init like:

    ${CEPH_BUILD_VIRTUALENVS}/ceph-detect-init/